index.js

```diff
@@ -1,9 +1,15 @@
 'use strict';
 
+const kerberos = require('./lib/kerberos');
+
 // Get the Kerberos library
-module.exports = require('./lib/kerberos');
+module.exports = kerberos;
+
+// Support legacy versions of the mongodb driver which expect this export
+module.exports.Kerberos = kerberos;
```

Comment:
Optional: This circular reference will probably be bad for future module imports.Can we either:

A) deprecate this and come up with a plan for removing it?
B) make a ticket for doing `A`?

Reply:
I thought about this a bit last night. Unfortunately, this is the only way to do this without requiring users to update their driver. At the same time it's frustrating to make this change in a major release, as we will have to live with it. I created [NODE-1600](https://jira.mongodb.org/browse/NODE-1600) to deal with this.